dist/index.js

```diff
@@ -4799,9 +4799,11 @@ class GitCommandManager {
     branchList(remote) {
         return __awaiter(this, void 0, void 0, function* () {
             const result = [];
-            // Note, this implementation uses "rev-parse --symbolic" because the output from
+            // Note, this implementation uses "rev-parse --symbolic-full-name" because the output from
             // "branch --list" is more difficult when in a detached HEAD state.
-            const args = ['rev-parse', '--symbolic'];
+            // Note, this implementation uses "rev-parse --symbolic-full-name" because there is a bug
```

Comment:
i ran into this bug in git 2.18 when testing. Switching to `--symbolic-full-name` and trimming `refs/heads/` and `refs/remotes/` from the output works around the issue.